behat.yml.dist

```diff
@@ -26,7 +26,7 @@ default:
             files_path: "%paths.base%/src/Sylius/Behat/Resources/fixtures/"
             base_url: "https://127.0.0.1:8080/"
             default_session: symfony
-            javascript_session: panther
+            javascript_session: chromedriver
```

Comment:
Something bad happened, and for unknown reason GitHub Actions can't run Chrome on the CI. Let's go with switching to the `chromedriver` for now, and trying to fix it later.